src/http/websocket_http_client.zig

```diff
@@ -86,22 +89,25 @@ fn buildRequestBody(
         host_.deinit();
     }
 
+    const host_fmt = ZigURL.HostFormatter{
+        .is_https = is_https,
+        .host = host_.slice(),
+        .port = port,
+    };
     const headers_ = static_headers[0 .. 1 + @as(usize, @intFromBool(client_protocol.len > 0))];
     const pico_headers = PicoHTTP.Headers{ .headers = headers_ };
 
     return try std.fmt.allocPrint(
         allocator,
         "GET {s} HTTP/1.1\r\n" ++
-            "Host: {s}\r\n" ++
-            "Pragma: no-cache\r\n" ++
```

Comment:
why get rid of these?

Reply:
> Other clients don't send them, they don't affect servers because the request is upgrade (non-cacheable), and it could conflict with user-set headers. 

Host header is a must for 1.1 but not for 1.0, we should keep at least Host

```
A client MUST include a Host header field in all HTTP/1.1 request messages
```
https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

Reply:
`Host` is included? Check the diff

Reply:
> `Host` is included? Check the full diff

Oh sorry is included in fact.